src/stages/stage5/timeline.c

```diff
@@ -400,17 +366,114 @@ static int stage5_superbullet(Enemy *e, int t) {
 	return 1;
 }
 
-void stage5_events(void) {
-	TIMER(&global.timer);
+TASK(greeter_fairy_attack, {
+		BoxedEnemy e;
+		int count;
+}) {
+	Enemy *e = TASK_BIND(ARGS.e);
+	cmplx dir = cnormalize(global.plr.pos - e->pos);
+
+	for(int i = -ARGS.count; i <= ARGS.count; i++) {
+		PROJECTILE(
+			.proto = pp_bullet,
+			.pos = e->pos,
+			.color = RGB(0.0, 0.0, 1.0),
+			.move = move_asymptotic_simple(3.5 * cdir(carg(dir) + 0.06 * i), 5),
```

Comment:
It's generally more efficient and cleaner to do `dir * cdir(X)` instead of `cdir(carg(dir) + X)`.

Also in the original code, the bullet speed is 4.5 on Lunatic (that's the `+(global.diff == D_Lunatic)` term). I suggest defining a `real speed = difficulty_value(3.5, 3.5, 3.5, 4.5);` outside the loop